pkg/server/share_manager.go

```diff
@@ -247,7 +248,7 @@ func (m *ShareManager) hasHealthyVolume() error {
 	mounter := mount.New("")
 	mountPoints, _ := mounter.List()
 	for _, mp := range mountPoints {
-		if mp.Path == mountPath && isMountPointReadOnly(mp) {
+		if mp.Path == mountPath && util.IsMountPointReadOnly(mp) {
```

Comment:
We should move `IsMountPointReadOnly` from this package as well.

Reply:
Updated. Thank you.